dataland-frontend/src/components/resources/dataTable/conversion/DataPoints.ts

```diff
@@ -43,11 +44,21 @@ export function getDataPointGetterFactory<
     const formattedValue = formatter(dataPoint);
     let displayValue: string;
     if (formattedValue == undefined || formattedValue == "") {
-      displayValue = "No data provided";
+      displayValue = NO_DATA_PROVIDED;
     } else {
       displayValue = formattedValue;
     }
     const dataPointAsExtendedDataPoint = dataPoint as unknown as ExtendedDataPoint<V>;
+    if (
```

Comment:
Why was that added? I think this is already covered by the else case in the if below?

Reply:
Checked would look ok for the prod data